config/routes.rb

```diff
@@ -85,5 +83,8 @@
     get '/tasks'    => 'profiles#host_tasks', as: :host_tasks
   end
 
-  get '/:id' => 'cities#forbes_show', as: :forbes_city
+  # This has to be at the bottom of the file or every
+  # route that is not the root route will match this
+  # and be routed to CitiesController.
+  get '/:id' => 'cities#show', as: :forbes_city
```

Comment:
This is bad practice. This will catch every route that is not root and is not captured in the rest of the file. I'm not changing in this PR, but aded a note. Also, the CitiesController#show throws an error when you go `/blahblah` becuase it's trying to look up a city based on `blabblah`, so that's a bug anyway.